### ISSUE TITLE
Not recognizing package

### ISSUE THREAD

**Issue by paulrobertpine** [martijnversluis/ChordSheetJS#66]
I'm now getting an error in my code

```
 ERROR  Failed to compile with 1 errors                                                                               

This dependency was not found:

* chordsheetjs in ./src/pages/index.js

To install it, you can run: npm install --save chordsheetjs                                                                                                

```

This is **after** I've installed ChordSheetJS; I can see it in the node_modules folder.

It looks like the /lib folder is missing from the npm package

**Comment by martijnversluis** [comment]
@paulrobertpine Thanks for reporting, I'll try to check/reproduce.

**Comment by martijnversluis** [comment]
@paulrobertpine Fixed for now. Turns out the automatic deployment doesn't quite work. I manually released 2.8.1.
